portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Contact</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body, class="contact">
```

Comment:
Why a comma after the body? Options: a) obscure HTML syntax I am not aware of OR b) typo